codegen/weave-device-cpp/templates/event_struct.cpp.inc

```diff
@@ -64,4 +64,12 @@ const nl::SchemaFieldDescriptor {{struct.base_name}}::FieldSchema =
     .mFields = {{struct.base_name}}FieldDescriptors,
     .mSize = sizeof({{struct.base_name}})
 };
+{% else %}
```

Comment:
Gratuitous, unrelated change?

Reply:
Three commits tackling two unrelated issues.  I had named the PR to be descriptive of both changes, and the commit messages are meaningful and relate to the right sections of the functionality.  Unfortunately, the resulting PR ends up ignoring the commit messages in the top-level description.  I will split this into different PRs.

Reply:
Should be resolved now.